pkg/mdformatter/mdgen/testdata/mdgen_not_formatted.md

```diff
@@ -37,4 +37,7 @@ alertmanagers:
 
 ```bash mdox-expect-exit-code=2 mdox-gen-exec="bash ./testdata/out3.sh"
 abc
-```
\ No newline at end of file
+```
+
+```bash mdox-gen-exec="sed -n '1,3p' ./testdata/out3.sh"
```

Comment:
Since we actually need only exec, wonder, do you think just `mdox=` or `mdox-exec=` would do the work? No need for `gen` then? (: